logistics/scripts/modules/database/postgres-database-management.psm1

```
@@ -5,42 +5,44 @@
 $script:packageName = "PostgreSQL.Binaries"
 $script:packageVersion = "13.7.1"
+$script:toolsPath = (Get-ToolsPath)
+$script:windowsPostgreSQLBinariesPath = "$script:toolsPath/$script:packageName.$script:packageVersion/tools/"
 
 function Test-PostgreSQLBinariesInstalled {
-    if (!(Get-IsWindows)) {
-
-        return IsCommandAvailable "psql"
-    }
 
-    $packagePath = "$script:toolsPath/$script:packageName.$script:packageVersion"
+    $testerCommand = @(If (Get-IsWindows) { "Test-Path" } Else { "IsCommandAvailable" })
 
-    $psqlExists = (Test-Path "$packagePath/tools/psql.exe")
-    $pgdumpExists = (Test-Path "$packagePath/tools/pg_dump.exe")
-    $pgrestoreExists = (Test-Path "$packagePath/tools/pg_restore.exe")
+    $psqlExists = & $testerCommand (Get-PSQLPath)
+    $pgdumpExists = & $testerCommand (Get-PGDumpPath)
+    $pgrestoreExists = & $testerCommand (Get-PGRestorePath)
 
     return ($psqlExists -and $pgdumpExists -and $pgrestoreExists)
 }
 
-function Get-PostgreSQLBinariesPath { return (Resolve-path "$script:toolsPath/$script:packageName.$script:packageVersion/tools/").Path }
+function Get-PSQLPath { return @(If (Get-IsWindows) { Join-Path $windowsPostgreSQLBinariesPath "psql.exe" } Else { "psql" }) }
 
-function Get-PSQLPath { return (Resolve-path (Join-Path (Get-PostgreSQLBinariesPath) 'psql.exe')).Path }
+function Get-PGDumpPath { return @(If (Get-IsWindows) { Join-Path $windowsPostgreSQLBinariesPath "pg_dump.exe" } Else { "pg_dump" }) }
 
-function Get-PGDumpPath { return (Resolve-path (Join-Path (Get-PostgreSQLBinariesPath) 'pg_dump.exe')).Path }
-
-function Get-PGRestorePath { return (Resolve-path (Join-Path (Get-PostgreSQLBinariesPath) 'pg_restore.exe')).Path }
+function Get-PGRestorePath { return @(If (Get-IsWindows) { Join-Path $windowsPostgreSQLBinariesPath "pg_restore.exe" } Else { "pg_restore" }) }
 
 function Install-PostgreSQLBinaries {
+    if(!(Get-IsWindows)) {
```

Moved this `if` up so that we fail fast in Linux.